app/views/nominations/already_nominated.html.erb

```diff
@@ -3,24 +3,15 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-header-l">
-      You can only send 1 nomination email per day
+      An Induction Tutor has already been nominated for your school
     </h1>
-    <p>
-      You have already sent a renomination email today. Try again tomorrow.
-    </p>
 
     <h2 class="govuk-heading-m">
-      If you haven't received this email
+      What to do next
     </h2>
     <ul class="govuk-list govuk-list--bullet">
-      <li>
-        we cannot disclose the email address. It will be an administration email, for example office@, admin@ or head@
-      </li>
-      <li>
-        if your school does not receive this email, ensure your schools email address is correct on our register,
-        <%= govuk_link_to "Get information about schools", "https://www.get-information-schools.service.gov.uk/" %>
-      </li>
-      <li>if your school does not receive this email, contact: <%= render MailToSupportComponent.new %></li>
+      <li>Ask your colleagues about it</li>
+      <li>Contact DfE on <%= render MailToSupportComponent.new %></li>
```

Comment:
same here! (see what's being removed for example of how it should be)

Reply:
It shows as removed because I've renamed the original to already_renominated - so that changed the same. Miro board shows the text in specific format and unless that will change there - I will use it as source of reference

Reply:
<img width="554" alt="Screenshot 2021-04-01 at 13 55 29" src="https://user-images.githubusercontent.com/78035269/113297310-5cb46800-92f2-11eb-8037-b69fa4069937.png">

Reply:
This is wrong. See content doc